tests/screenshots/readme.md

```diff
@@ -0,0 +1,3 @@
+Please do not add unecessary files to this directory. This is reserved for auto-generated screenshots from testing. 
+
+Please do not move this file.
```

Comment:
Do you want to include the images in this PR? Or is it just used for debugging? If it's just used for debugging, could you rename this folder like "debugging-snapshots" to make that clear?

Reply:
Oh NVM, I see, you're saving images in here and then taking a percy snapshots of those images. Clever 😼

Reply:
Updating it to make it clearer.